sql-cloudant/src/test/scala/org/apache/bahir/cloudant/CloudantSparkSQLSuite.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.bahir.cloudant
+
+import org.apache.spark.sql.{SparkSession, SQLContext, SQLImplicits}
+
+class CloudantSparkSQLSuite extends ClientSparkFunSuite {
+  // import spark implicits
+  private object testImplicits extends SQLImplicits {
+    protected override def _sqlContext: SQLContext = spark.sqlContext
+  }
+
+  import testImplicits._
+
+  val apiReceiver = "_all_docs"
+
+  override def beforeAll() {
+    runIfTestsEnabled("Prepare Cloudant test databases") {
+      super.beforeAll()
+      spark = SparkSession.builder().config(conf)
+        .config("cloudant.protocol", TestUtils.getProtocol)
+        .config("cloudant.host", TestUtils.getHost)
+        .config("cloudant.username", TestUtils.getUsername)
+        .config("cloudant.password", TestUtils.getPassword)
+        .config("cloudant.apiReceiver", apiReceiver)
```

Comment:
@mayya-sharipova It's `endpoint`, looks like I missed this test suite.  Thanks for catching that.